rules/sort-svelte-attributes.ts

```diff
@@ -124,7 +124,7 @@ export default createEslintRule<Options<string[]>, MESSAGE_ID>({
                 if (typeof attribute.key.name === 'string') {
                   ;({ name } = attribute.key)
                 } else {
-                  name = context.sourceCode.text.slice(...attribute.key.range!)
+                  name = context.sourceCode.text.slice(...attribute.key.range)
```

Comment:
Fixing the issue due to an ESLint error.
```
 ✝  study/fork/eslint-plugin-perfectionist   main±  node_modules/.bin/eslint "**/*.{js,ts}"

/Users/ryo.takahashi/git/study/fork/eslint-plugin-perfectionist/rules/sort-svelte-attributes.ts
  127:59  error  This assertion is unnecessary since it does not change the type of the expression  @typescript-eslint/no-unnecessary-type-assertion

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```